src/components/icon-button/icon-button.style.js

```diff
@@ -1,12 +1,13 @@
 import styled, { css } from "styled-components";
+import { space } from "styled-system";
```

Comment:
```suggestion
import { margin } from "styled-system";
```

We decided in the Jira ticket to only add margin options to this component